pkgs/by-name/pl/plasticscm-client-gui/package.nix

```
@@ -0,0 +1,102 @@
+{
+  buildFHSEnv,
+  extraLibs ? _: [ ],
+  extraPkgs ? _: [ ],
+  imagemagick,
+  lib,
+  makeWrapper,
+  plasticscm-client-core-unwrapped,
+  plasticscm-client-gui-unwrapped,
+  plasticscm-theme,
+}:
+assert plasticscm-client-gui-unwrapped.version == plasticscm-client-core-unwrapped.version;
+assert plasticscm-client-gui-unwrapped.version == plasticscm-theme.version;
+buildFHSEnv {
+  pname = "plasticscm-client-gui";
+  inherit (plasticscm-client-gui-unwrapped) version meta;
+
+  runScript = "";
+
+  targetPkgs =
+    pkgs:
+    with pkgs;
+    [
+      plasticscm-client-gui-unwrapped
+
+      # Dependencies from the Debian package
+      plasticscm-client-core-unwrapped
+
+      # Undocumented dependencies discovered from testing
```

```suggestion
      plasticscm-client-core-unwrapped

```
fontconfig is more a nixos issue as that just exists on Debian like systems

---

No, just the comments are more misleading than they help.